public/javascripts/itemDetailView.js

```diff
@@ -24,11 +31,9 @@ var itemDetailView = {
             } else {
                 self.renderError("No inventory found");
             }
-        })/*.fail(function (response) {
-            self.renderError("Failed to load inventory: " + response);
-        })*/;
+        });
 
-        navigation.hit("/GetRunsByProduct/" + this.productID, function (response) {
+        navigation.get("/GetRunsByProduct/" + this.productID, function (err, response) {
```

Comment:
That check should probably be more robust, but it works for our cases atm.
Cases:
http://env.thisisimp.com/api/route
env.thisisimp.com/api/route
/route

If I were to give say:
route+env.thisisimp.com/api/
The check would turn it into:
env.thisisimp.com/api/route/env.thisisimp.com/api/
But in that case, everything was borked to begin with, so making it more borked does not matter.